src/test/scala/org/broadinstitute/dsde/firecloud/model/DuosModelSpec.scala

```diff
@@ -0,0 +1,266 @@
+        duosDataUse.sexualDiseases.getOrElse(false) shouldBe true
+      }
+      "stigmatizeDiseases: true" in {
+        val jsValues: Map[String, JsValue] = Map("stigmatizeDiseases" -> JsBoolean(true))
+        val duosDataUse: DuosDataUse = DuosDataUse.apply(jsValues)
+        duosDataUse.stigmatizeDiseases.getOrElse(false) shouldBe true
+      }
+      "vulnerablePopulations: true" in {
+        val jsValues: Map[String, JsValue] = Map("vulnerablePopulations" -> JsBoolean(true))
+        val duosDataUse: DuosDataUse = DuosDataUse.apply(jsValues)
+        duosDataUse.vulnerablePopulations.getOrElse(false) shouldBe true
+      }
+      "psychologicalTraits: true" in {
+        val jsValues: Map[String, JsValue] = Map("psychologicalTraits" -> JsBoolean(true))
+        val duosDataUse: DuosDataUse = DuosDataUse.apply(jsValues)
+        duosDataUse.psychologicalTraits.getOrElse(false) shouldBe true
+      }
+      "nonBiomedical: true" in {
+        val jsValues: Map[String, JsValue] = Map("nonBiomedical" -> JsBoolean(true))
+        val duosDataUse: DuosDataUse = DuosDataUse.apply(jsValues)
+        duosDataUse.nonBiomedical.getOrElse(false) shouldBe true
+      }
+    }
+  }
+
+  private def assertIsUndefined(duosDataUse: DuosDataUse): Unit = {
+    duosDataUse.generalUse.isDefined shouldBe false
+    duosDataUse.hmbResearch.isDefined shouldBe false
+    duosDataUse.diseaseRestrictions.isDefined shouldBe false
+    duosDataUse.populationOriginsAncestry.isDefined shouldBe false
+    duosDataUse.populationStructure.isDefined shouldBe false
+    duosDataUse.commercialUse.isDefined shouldBe false
+    duosDataUse.methodsResearch.isDefined shouldBe false
+    duosDataUse.aggregateResearch.isDefined shouldBe false
+    duosDataUse.controlSetOption.isDefined shouldBe false
+    duosDataUse.gender.isDefined shouldBe false
+    duosDataUse.pediatric.isDefined shouldBe false
+    duosDataUse.populationRestrictions.isDefined shouldBe false
+    duosDataUse.dateRestriction.isDefined shouldBe false
+    duosDataUse.recontactingDataSubjects.isDefined shouldBe false
+    duosDataUse.recontactMay.isDefined shouldBe false
+    duosDataUse.recontactMust.isDefined shouldBe false
+    duosDataUse.genomicPhenotypicData.isDefined shouldBe false
+    duosDataUse.otherRestrictions.isDefined shouldBe false
+    duosDataUse.cloudStorage.isDefined shouldBe false
+    duosDataUse.ethicsApprovalRequired.isDefined shouldBe false
+    duosDataUse.geographicalRestrictions.isDefined shouldBe false
+    duosDataUse.other.isDefined shouldBe false
+    duosDataUse.illegalBehavior.isDefined shouldBe false
+    duosDataUse.addiction.isDefined shouldBe false
+    duosDataUse.sexualDiseases.isDefined shouldBe false
+    duosDataUse.stigmatizeDiseases.isDefined shouldBe false
+    duosDataUse.vulnerablePopulations.isDefined shouldBe false
+    duosDataUse.psychologicalTraits.isDefined shouldBe false
+    duosDataUse.nonBiomedical.isDefined shouldBe false
+  }
```

Comment:
can you use the same reflection pattern here that you used in the json deserialization?
```
  private def assertIsUndefined(duosDataUse: DuosDataUse): Unit = {
    duosDataUse.getClass.getDeclaredFields map { f =>
      f.setAccessible(true)
      f.get(duosDataUse) match {
        case Some(x) => fail(s"field ${f.getName} should be None; it is $x")
        case _ => // noop
      }
    }
}
```